src/components/side-panel/interaction-panel/tooltip-config/constants.js

```diff
@@ -0,0 +1,81 @@
+export const TOOLTIP_FORMAT_TYPES = {
```

Comment:
can you move this constant file to to `src/constants/tooltip.js` instead of nesting it in the components folder?

Reply:
Sure